p_set_1.rb

```diff
@@ -2,17 +2,48 @@
 # SMART WATER
 BOTTLE_VOLUME = 16
 WATER_FLOW_RATE = 192
+
 def bottle_reporter
-  puts "That's #{bottle_calculator get_minutes} bottles of water"
+  puts "That's #{bottle_calculator ask_minutes} bottles of water"
 end
 
 def bottle_calculator(minutes)
   minutes * WATER_FLOW_RATE / BOTTLE_VOLUME
 end
 
-def get_minutes
-  puts "How many minutes do you shower for?"
+def ask_minutes
+  puts "How many mins do you shower for?"
   minutes = gets.chomp.to_i
 end
 
+# ITSA MARIO
+SPACE =      " "
+BRICKS =     "#"
+APEX_WIDTH = 2
```

Comment:
It's a small thing, but I personally prefer aligned equal signs as well as aligning the right-hand-side.